Orchestrator.py

```diff
@@ -826,6 +844,50 @@ def snsInit(self):
 		sns_workload	= self.workloadSpecificationDict[Orchestrator.WORKLOAD_SPEC_PARTITION_KEY]
 		self.sns	= SnsNotifier(sns_topic_name,sns_workload)
 
+	@retriable(attempts=5, sleeptime=0, jitter=0)
+	def readWorkloadStateTable(self):
+		try:
+			dynamodbWorkloadState = self.WorkloadStateTable.query(
+				KeyConditionExpression=Key('Workload').eq(self.partitionTargetValue),
+				ConsistentRead=False,
+				ReturnConsumedCapacity="TOTAL",
+			)
+		except ClientError as e:
+			logger.error('Exception encountered in readWorkloadStateTable() -->' + str(e))
+
+		return dynamodbWorkloadState['Items'][0]['LastActionType']
+
+	def updateWorkloadStateTable(self,action):
+
+		self.currentTime = str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+
+		if (action != Orchestrator.ACTION_STOP) and (self.scalingProfile is not None):
```

Comment:
Let's change to 
if (action == Orchestrator.ACTION_START) and (self.scalingProfile is not None):